lang/C.md

```diff
@@ -0,0 +1,112 @@
+
+## Setting up C environment locally
+
+**Step 1:**
+
+You have to download a text editor or an IDE to type your code. Examples include Notepad++,Sublime,vim. An integrated development environment or an IDE is a software application that provides comprehensive facilities to computer programmers for software development. Examples include Visual Studio Code, CLion.
+
+- Sublime - Navigate to [https://www.sublimetext.com/3](https://www.sublimetext.com/3) to download the latest stable version of Sublime according to your OS (Windows/Linux/OSX).
+- Visual Studio Code - Navigate to [https://code.visualstudio.com/download](https://code.visualstudio.com/download)  to download the latest stable version of Sublime according to your OS (Windows/Linux/OSX).
+
+**Step 2:**
+
+Install the Compiler according to your OS
+
+#### Linux   
+
+If you are using Linux , then check whether GCC is installed on your system by entering the following command from the command line - `gcc --version`
+
+If you have GNU compiler installed on your machine, then it should print a message as follows:
+
+"gcc (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0 Copyright (C) 2019 Free Software Foundation, Inc. 
+This is free software; see the source for copying conditions.  There is NO
+warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE."
+
+If it is not installed you can install using these commands:
+
+`sudo apt-get install gcc`
+`sudo apt-get install build-essential`
+
+#### Windows
+
+Get the latest version of Mingw-w64 via [MSYS2](https://www.msys2.org/), which provides up-to-date native builds of GCC, Mingw-w64, and other helpful C++ tools and libraries. Then follow the instructions on the [MSYS2](https://www.msys2.org/) website to install Mingw-w64.
+
+Add the path to your Mingw-w64 `bin` folder to the Windows `PATH` environment variable by using the following steps:
+
+- In the Windows search bar, type 'settings' to open your Windows Settings.
+- Search for Edit environment variables for your account.
+- Choose the Path variable and then select Edit.
+- Select New and add the Mingw-w64 destination folder path to the system path. The exact path depends on which version of Mingw-w64 you have installed and where you installed it. If you used the settings above to install Mingw-w64, then add this to the path: C:\msys64\mingw64\bin.
+- Select OK to save the updated PATH. You will need to reopen any console windows for the new PATH location to be available
+
+
+**Check your MinGW installation**
+
+To check that your Mingw-w64 tools are correctly installed and available, open a new Command Prompt and type:
+
+- `g++ --version`
```

Comment:
Again, should be surrounded by 3 backticks followed by the language name & a line break.

Reply:
I think it's OK to keep it as-is as there are two available options, and IMO it looks better like this. 🙂